recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "segmentation-mask-overlay" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lobantseff/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4d33eebc433b350d57cc0d6154a3b4112f886666f5acb8ecddab5b67286d4476
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # Unsatisfiable dependencies for platform linux-s390x: {MatchSpec("opencv[version='>=4.5.3']")}
+  skip: True  # [s390x]
+  # Not avaliable opencv on py313 
+  skip: True  # [py==313]
```

```suggestion
  skip: True  # [py>=313]
```

Would cover us in October when 3.14 comes out!

---

done